pandas/tests/groupby/test_groupby.py

```diff
@@ -1825,6 +1825,18 @@ def test_groupby_empty_list_raises():
         df.groupby([[]])
 
 
+def test_groupby_with_empty_category():
```

Comment:
can you move this to groupby/test_categorical.py

Reply:
Done, thanks!